engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/views/errata.html

```diff
@@ -2,6 +2,14 @@
 
 <div data-extend-template="layouts/nutupane.html">
 
+  <div data-block="messages">
+    <div bst-alert="info" ng-show="incrementalUpdateInProgress">
+      <p translate>
+        There is currently an Incremental Update in progress.  This update must finish before applying existing updates.
+      </p>
```

Comment:
I would link to the first task returned from the tasks api.  That would help the user go see what is going on.